terraform/vars/terraform-dev.tfvars

```diff
@@ -10,6 +10,7 @@ JWT_SECRET        = "@@JWT_SECRET@@"
 redis_queue_name  = "mail"
 APP_URL           = "https://gfw-web-dev.cube-cdn.com"
 USERS_API_URL     = "https://staging-api.resourcewatch.org/v1"
+AREAS_API_URL     = "https://dev-fw-api.globalforestwatch.org/v3/forest-watcher"
```

Comment:
is it possible to change this so it uses `https://${data.terraform_remote_state.fw_core.outputs.public_url}/v3/forest-watcher` within main.tf?

See comment code within fw_api: https://github.com/wri/fw_api/blob/73d67c7caf7d998b761bc7adfa6fa79ccb71ef4a/terraform/main.tf#L84